public/static/app/perf/trend_chart.js

```diff
@@ -899,16 +909,24 @@ mciModule.factory('DrawPerfTrendChart', function (
     //    The segment position is outside of domain, pos = 0
     // C. domain=[0, 100], pos=99, len=10
     //    The segment end position is outside of domain, pos = 90
-    function fit1d(domain, pos, len) {
-      return pos < domain[0]       ? 0 :
-             pos + len > domain[1] ? domain[1] - len :
-             pos
+    function fit1d(domain, pos, len, noOverlap) {
```

Comment:
Do we need to parameterize noOverlap? Under what situations would we set this to true?